dashboard/cron.py

```diff
@@ -263,10 +273,15 @@ def update_with_bq_access(self):
                 bigquery.ArrayQueryParameter('course_ids_short', 'STRING', data_warehouse_course_ids_short),
                 bigquery.ScalarQueryParameter('canvas_data_id_increment', 'INT64', settings.CANVAS_DATA_ID_INCREMENT)
             ]
+            if (course_start_time is not None):
+                # insert the start time parameter for query
+                query_params.append(bigquery.ScalarQueryParameter('course_start_time', 'TIMESTAMP', course_start_time))
+
             job_config = bigquery.QueryJobConfig()
             job_config.query_parameters = query_params
 
             # Location must match that of the dataset(s) referenced in the query.
+            logger.debug(final_bq_query)
```

Comment:
It looks like `final_bq_query` will be the same here as when it is logged above `query_params`. Can we remove one of these?